src/js/services/auth-service.js

```diff
@@ -0,0 +1,11 @@
+import { store } from 'stores/app-store';
+import { core } from 'services/core';
+
+import { setUser } from 'actions/user-actions';
+
+export function login(props) {
+    return core().appUsers.init(props).catch((e) => {
+        console.log(e); //eslint-disable-line no-console
```

Comment:
It's pretty much up to use.. I don't want people to complain because it fails silently.. and since everything is handled in promises now, you have to catch it to know what failed.